.github/workflows/update_protos.yml

```diff
@@ -14,6 +14,8 @@ jobs:
       image: ghcr.io/cirruslabs/flutter:latest
     steps:
       - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.GIT_ACCESS_TOKEN }}
```

Comment:
Other PRs use repo read token?

Reply:
Yeah, flutter doesn't have access to the `REPO_READ_TOKEN`, but 1) they should have approximately the same permissions, and 2) I'm not actually sure this part is even necessary. It's a bit surprising to me that this shows up as a diff at all here because this part was already merged in when this branch was merged the other day. But either way, the most important change is changing the token of the Open PR step, as it changes _who_ opens the PR from `git-actions` to `viambot` and therefore changes the capacity to run tests.